manifests/charts/istio-control/istio-discovery/files/gen-istio.yaml

```diff
@@ -45,7 +45,7 @@ data:
         zipkin:
           address: zipkin.istio-system:9411
     enablePrometheusMerge: true
-    rootNamespace: istio-system
+    rootNamespace: ""
```

Comment:
Won't this break everyone? its going to set it to empty string now?

Reply:
Resolved.